src/aws/lambda-encode/src/updateState.js

```diff
@@ -1,4 +1,4 @@
-// DUPLICATE CODE: terraform/source/update_state/src/updateState
+// DUPLICATE CODE: src/aws/lambda-update-state/src/updateState
```

Comment:
Just occured to me that `lambda-update-state` may be badly named. It would better not share name with a function of the `lambda-encode`? Maybe `lambda-result` is a better name?

Reply:
#147